src/main/java/org/embulk/output/DatabricksOutputPlugin.java

```diff
@@ -92,6 +107,14 @@ protected JdbcOutputConnector getConnector(PluginTask task, boolean retryableMet
     props.put("ConnCatalog", t.getCatalogName());
     props.put("ConnSchema", t.getSchemaName());
     props.putAll(t.getOptions());
+    // overwrite UserAgentEntry property if the same property is set in options
+    if (t.getUserAgentEntry() != null) {
```

Comment:
`t.getUserAgentEntry` always seems to be not null, May I suggest that you can delete this line.

Reply:
The current code did not guarantee the behavior of 2 below, so changed condition.

- 1: options is not set & user-agent is not set => unknown/0.0.0
- 2: options is set & user-agent is not set => value of options
- 3: options is not set & user-agent is set => value of user-agent
- 4: options is set, user-agent is set => value of user-agent

Reply:
We have decided to always overwrite the UserAgentEntry value in options by user-agent, so please ignore the above comment. I removed condition as you commented.